test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs

```diff
@@ -109,144 +149,142 @@ public void WinTouchEvent(object sender, Window.TouchEventArgs e)
                         break;
 
-                    case "4":
-                        TestCase4();
+                    case key4:
+                        TestCreateSubWindowSecondWhichHasSomeSettingAndDisposeItAfterThreeSecondsDelay();
                         break;
 
-                    case "5":
-                        TestCase5();
+                    case key5:
+                        TestCreateSubWindowSecondWhichHasSomeSettingAndAddSomeDummyObjectAndDisposeItAfterThreeSecondsDelay();
                         break;
 
-                    case "6":
+                    case key6:
                         SetParentAbove();
                         break;
 
-                    case "7":
+                    case key7:
                         SetParentBelow();
                         break;
 
                     default:
-                        log.Fatal(tag, $"no test!");
+                        log.Debug(tag, $"no test!");
                         break;
                 }
             }
         }
 
         //TDD
-        void TestCase1()
+        private void TestMakeSubWindowAndAddSomeDummyObject()
         {
-            log.Fatal(tag, "test 1 : 1) make sub window-1 2) add some dummy object");
-
-            subWin1 = new Window("subwin1", new Rectangle(500, 500, 300, 300), false);
-            subWin1.BackgroundColor = Color.Blue;
+            subWinFirst = new Window("subWinFirst", new Rectangle(5 * testPosition, 5 * testPosition, 3 * testSize, 3 * testSize), false)
+            {
+                BackgroundColor = Color.Blue
+            };
             View dummy = new View()
             {
-                Size = new Size(100, 100),
-                Position = new Position(50, 50),
+                Size = new Size(testSize, testSize),
+                Position = new Position(testPosition, testPosition),
                 BackgroundColor = Color.Yellow,
             };
-            subWin1.Add(dummy);
-            subWin1.KeyEvent += SubWin1_KeyEvent;
+            subWinFirst.Add(dummy);
+            subWinFirst.KeyEvent += SubWinFirstKeyEvent;
         }
-        void TestCase2()
+
+        private void TestDisposeSubWindowFirstWhichWasCreatedInPrivousTestCase()
         {
-            log.Fatal(tag, "test 2 : 1) do dispose of sub window-1 created in #1");
-            subWin1?.Dispose();
+            subWinFirst?.Dispose();
         }
-        void TestCase3()
+
+        private void TestCreateSubWindowSecondWhichDoesNotHaveAnySettingAndDisposeItAfterThreeSecondsDelay()
         {
-            log.Fatal(tag, $"test 3 : 1) create sub window2 which doesn't have any setting 2) dispose it after 3 second delay");
-            subWin2 = null;
-            subWin2 = new Window();
-            tm = new Timer(3000);
-            tm.Tick += Tm_Tick;
-            tm.Start();
+            subWinSecond = null;
+            subWinSecond = new Window();
+            timer = new Timer(3 * oneSecond);
```

Comment:
not only the timer tick fuction name,
this timer variable name would be better to change disposeTimer or disposeSubWinSecondTimer

Reply:
Thank for review. I fixed it.